pkg/spec/cluster.go

```diff
@@ -193,6 +193,8 @@ type ClusterStatus struct {
 
 	// Size is the current size of the cluster
 	Size int `json:"size"`
+	// HealthySize is the number of pods passing the liveliness probe, HTTP GET on port 2379
```

Comment:
To be more general what about tracking the pods since we're already checking them?
`healthyPods []*v1.Pod`
Or just the names or endpoints? Your call.